hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/KeyValueStreamDataChannel.java

```diff
@@ -131,6 +131,13 @@ ReferenceCountedObject<ByteBuf> pollAll() {
         refs.forEach(ReferenceCountedObject::release);
       });
     }
+
+    void cleanUpAll() {
+      final int size = deque.size();
+      for (int i = 0; i < size; i++) {
+        Optional.ofNullable(poll()).ifPresent(ReferenceCountedObject::release);
+      }
+    }
```

Comment:
Let's call it `releaseAll()` and check `isEmpty()`.  Also, `poll()` won't return null.
```java
    void releaseAll() {
      while (!deque.isEmpty()) {
        poll().release();
      }
    }
```